src/modules/Packages.rb

```diff
@@ -2702,6 +2700,16 @@ def format_missing_resolvables(type, list)
         _("These items (%{type}) need to be selected to install: %{list}") % {type: type, list: list}
       end
     end
+
+    def fs_mountpoint(filesystem)
+      filesystem.mountpoints[0] || ""
```

Comment:
@aschnell Good question about btrfs. So far, `btrfs_fs.mountpoints[0]` returns the main mount point. But I'm not 100% sure if @aschnell is planning to change that in the future, when subvolumes module is improved.

If a volume is mounted to multiple places... it does the same it used to do before (in the target map era). The idea is to adapt the code with the minimal set of changes both in the code and in the behavior.